### DISCUSSION TITLE
Can't get my github metrics

### DISCUSSION THREAD

**Discussion by lionelsamrat10** [lowlighter/metrics#713] [General]
When I am trying to fetch my metrics, it is showing like that: An error occurred while generating metrics :(
Internal Server Error: failed to process metrics correctly

**Comment by lowlighter** [comment]
Hi!
Seems you finally achieved to get your metrics according to #714

Note that this error on the web instance is mostly due to GitHub API timeout, or eventually lack of resources.
If you wish to have a reliable way of getting your metrics, it's advised to use the GitHub actions (which will be dedicated to you)

Thanks for using metrics 🙂

**Comment by lionelsamrat10** [comment]
Yes finally I resolved it.
